app/controllers/users_controller.rb

```diff
@@ -497,6 +497,10 @@ def password_reset
           Invite.invalidate_for_email(@user.email) # invite link can't be used to log in anymore
           secure_session["password-#{token}"] = nil
           secure_session["second-factor-#{token}"] = nil
+          UserHistory.create!(
+            target_user: @user, acting_user: @user,
```

Comment:
each one should be on a different line, this  kind of formatting is error prone in the future

Reply:
:+1: can we add this to Rubocop? - seems a waste for us humans to have to deal with this ;)

Reply:
tgx is working on having prettier working in ruby I think, so we should just wait for this